contracts/ERC20/ERC20ProxyStorage.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.5.13;
+
+/**
+ * All storage must be declared here
+ * New storage must be appended to the end
+ * Never remove items from this list
+ */
+contract ERC20ProxyStorage {
+
+    uint256 constant MAX_SUPPLY = 145000000000000000;
```

Comment:
@hal909 why do we need this when we also have the same thing in the other proxy storage contract?